install/docker/centos-docker-install.md

```diff
@@ -0,0 +1,44 @@
+# Installing Docker for Kata Containers on CentOS
+
+> **Note:**
+>
+> - This guide assumes you have
+>   [already installed the Kata Containers packages](../centos-installation-guide.md).
+
+1. Install the latest version of Docker with the following commands:
```

Comment:
I know  that we did not have issues recently with docker but do you think we need to ask to check versions.toml to verify the last known working/recommend  version.

Reply:
That's a good question, but I'm not sure how we can add in that check without a bunch of code (as we'd have to remove the leading `v` from https://github.com/kata-containers/runtime/blob/master/versions.yaml#L130. But we'd also have the complication of documenting in the install guides how to check the version of docker that the distro-specific package manager is offering to install and then we've got to have shell logic to compare the semantic versions.

However, we *could* conceivably turn the problem round - add an extra check at the *end* of https://github.com/kata-containers/documentation/blob/master/.ci/test-install-docs.sh where we check the output of `docker version` after we've successfully created a container. If the `major.minor` part of the semver version string does not match `yq read versions.yaml externals.docker.version`, we could fail the test since it indicates `versions.yaml` needs to be updated.

I'm not sure that's a valid test in the sense that that new test would actually be testing `versions.yaml` (in a different repo).

wdyt @grahamwhaley, @chavafg?